basic/exercise7/script7.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+function calculateDaysUntilToNextChristmas() {
+
+    var oneDay = 1000 * 60 * 60 * 24;
```

Comment:
It should be called in a way to specify what it is exactly - as I understand it is one day in miliseconds, so it should be called oneDayInMiliseconds